### PR TITLE
@mzikherman - Extract annotation into its own template, add link to based on text

### LINKED ISSUE #631: [Home] Related Artist rail: Link artist name in "based on" text
image 2017-01-03 at 4 54 05 pm

### PR DESCRIPTION
**Pull request by broskoski**
Closes #631 

screenshot 2017-01-05 11 16 52

### CODE REVIEW

File: apps/home/components/related_artists_context/annotation.jade [hunk 1]
```diff
@@ -0,0 +1 @@
+.abrv-follow-annotation.avant-garde-s-headline Based on #[a(href=based_on.href) #{based_on.name}]
```
Comment (mzikherman) [inline apps/home/components/related_artists_context/annotation.jade]: lol what are the brackets here for? Otherwise looks great, but me confused 🤔

Comment (broskoski) [inline apps/home/components/related_artists_context/annotation.jade]: Inline tags with jade :)

Comment (broskoski) [inline apps/home/components/related_artists_context/annotation.jade]: https://stackoverflow.com/questions/10953326/what-is-a-concise-way-to-create-inline-elements-in-jade

Comment (mzikherman) [inline apps/home/components/related_artists_context/annotation.jade]: sheesh!

**Pull request merged**